app/uk/gov/hmrc/nationalinsurancerecord/controllers/ErrorHandling.scala

```diff
@@ -33,9 +32,7 @@ trait ErrorHandling {
 
   def errorWrapper(func: => Future[Result])(implicit hc: HeaderCarrier): Future[Result] = {
     func.recover {
-      case e: NotFoundException =>
-        NotFound(Json.toJson(ErrorNotFound))
-
+      case e: NotFoundException => NotFound(Json.toJson(ErrorNotFound))
```

Comment:
Suggestion

As e is not being used here it could be made _

Reply:
Done